### ISSUE TITLE
NuGet package fails to load often with the Roslyn solution

### ISSUE THREAD

**Issue by davkean** [NuGet/Home#1597]
NuGet package often fails to load with the Roslyn solution, I was under the impression it might have been my hacked up install at work, but I experienced the same thing with the same issue today at home with a clean RTM install.
1. In my case, the package failed to load when I right-click on the Solution and choose Rebuild

Looking in the activity log, I see:

```
  <entry>
    <record>788</record>
    <time>2015/10/16 22:13:39.447</time>
    <type>Error</type>
    <source>VisualStudio</source>
    <description>SetSite failed for package [NuGetPackage]</description>
    <guid>{5FCC8577-4FEB-4D04-AD72-D6C629B083CC}</guid>
    <hr>80131500</hr>
    <errorinfo>An exception was thrown while initializing part "NuGet.PackageManagement.VisualStudio.VSSettings".</errorinfo>
  </entry>
  <entry>
    <record>789</record>
    <time>2015/10/16 22:13:39.447</time>
    <type>Error</type>
    <source>VisualStudio</source>
    <description>End package load [NuGetPackage]</description>
    <guid>{5FCC8577-4FEB-4D04-AD72-D6C629B083CC}</guid>
    <hr>80131500</hr>
    <errorinfo>An exception was thrown while initializing part "NuGet.PackageManagement.VisualStudio.VSSettings".</errorinfo>
  </entry>
```

**Comment by emgarten** [comment]
This looks related to https://github.com/NuGet/Home/issues/1529

**Comment by davkean** [comment]
Yep, certainly does. I've been trying to get a debugger attached to catch it, but so far been unsuccessful. Difference here though, is I have a solution open.

**Comment by yishaigalatzer** [comment]
@davkean can you install 3.2.1-RC it solves at least on of these issues. You got to love the ActivityLog for this :)

**Comment by yishaigalatzer** [comment]
@davkean can you try this with our latest builds (or by F5-ing from nuget.client solution) ?

**Comment by yishaigalatzer** [comment]
We fixed quite a few of these isues in NuGet 3.3. Please open a new bug if you still see it in NuGet 3.3
